docs/book/src/reference/metrics.md

```diff
@@ -92,5 +111,31 @@ func init() {
 ```
 
 You may then record metrics to those collectors from any part of your
-reconcile loop, and those metrics will be available for prometheus or
+reconcile loop. These metrics can be evaluated from anywhere in the operator code.
+
+
+For example:
+```
+func (r *MemcachedReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	...
+	...
+	...
+
+	goobers.Inc()
+	gooberFailures.Inc()
+	
+	...
+	...
+	
+	return ctrl.Result{}, nil
+}
```

Comment:
```suggestion
```

Actually this example is not necessary. It may look confusing to increment `goobers` and `gooberFailers` in the same reconcile loop.